CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-20.1.2
+
+**Note**: The `linkerd-proxy` version included with this release is more
+experimental than usual. We'd love your help testing, but be aware that there
+might be stability issues.
+
+* CLI
+  * Added HA specific checks to `linkerd check` to ensure that the `kube-system`
+    namespace has the `config.linkerd.io/admission-webhooks:disabled`
+    label set
+  * Fixed a problem causing the presence of unnecessary fields in
```

Comment:
Maybe emphasize that it's _unnecessary **empty** fields_, so that users who version-control the Linkerd YAML know what to expect.